AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -512,6 +512,9 @@ - (void)_updateSectionContextsWithChangeSet:(_ASHierarchyChangeSet *)changeSet
     return;
   }
   
+  // TODO if the change set includes solely section reloads that together are equivalent to reloadData (i.e reload the only section),
+  // do a reloadData here as an optimization.
+  
```

Comment:
Since `reloadData` is never animated, but section reloads may be, we probably won't want to build this optimization.